src/react/deckgl.js

```diff
@@ -100,12 +108,18 @@ export default class DeckGL extends React.Component {
     this._updateLayers(this.props);
 
     // Check if a mouse event has been specified and that at least one of the layers is pickable
-    const hasEvent = this.props.onLayerClick !== noop || this.props.onLayerHover !== noop;
+    const hasEvent =
+      this.props.onLayerClick !== noop ||
+      this.props.onLayerHover !== noop ||
+      this.props.onLayerDragStart !== noop ||
+      this.props.onLayerDragMove !== noop ||
+      this.props.onLayerDragEnd !== noop ||
+      this.props.onLayerDragCancel !== noop;
```

Comment:
You're right @Pessimistress, this is an anti-pattern. However, it's only checking to offer a warning, in the case that an `onLayer*` handler is passed when no layers have set `pickable`; I imagine the overwhelmingly most common use case will be to set `onLayer*` handlers on init, not dynamically at some point later in the program. Checking these props on every update just to conditionally issue a warning doesn't seem worth the (probably minor) cost.

My vote is we leave as-is unless it proves to be a problem.